x-pack/plugins/security_solution/public/common/hooks/endpoint/ingest_enabled.ts

```diff
@@ -15,12 +15,12 @@ export const useIngestEnabledCheck = (): {
   write: boolean;
   read: boolean;
 } => {
-  const { services } = useKibana();
+  const { services } = useKibana<{ application: ApplicationStart }>();
```

Comment:
We've actually got our own [typed export](https://github.com/elastic/kibana/blob/master/x-pack/plugins/security_solution/public/common/lib/kibana/kibana_react.ts#L23) of `useKibana` that must've been overlooked here. No need to change it here, just pointing it out for posterity; we will update these in a subsequent PR.